app/src/main/java/com/example/recipeapp/DetailsActivity.java

```diff
@@ -89,5 +92,24 @@ public void onClick(View v) {
                 startActivity(i);
             }
         });
+
+        binding.options.like.setOnClickListener(this::onLike);
+    }
+
+    private void onLike(View v) {
+        Toast.makeText(DetailsActivity.this, R.string.liked, Toast.LENGTH_SHORT).show();
+        recipe.getTaste(new Callback<Taste>() {
```

Comment:
Nit , let’s move it to viewmodel

Reply:
I'm working on a ViewModel in a different branch, so I'll move it there!